code/modules/surgery/organs/blood.dm

```diff
@@ -202,6 +202,10 @@
 		blood_data["factions"] = faction
 		blood_data["dna"] = dna.Clone()
 		return blood_data
+	if(blood_id == "slimejelly")
```

Comment:
This currently won't do anything since all the code around it either uses the virus data and the like from above. Or `blood_color`.
You can try making slimejelly behave sort of the same as normal blood. Might also help with the creation of slime blood etc as normal blood uses the `blood_color` to make it and it works just fine

Reply:
removed